.github/ISSUE_TEMPLATE/z_release_patch.md

```diff
@@ -11,6 +11,7 @@ assignees: ''
 
 - [ ] Create an GH Milestone and add this issue to it.
 - [ ] Add all the issues to be worked on to the Milestone.
+- [ ] Ensure that all issues are labelled correctly, particularly ensure that "Regressions" are labelled with "Affects: <version>" labels.
```

Comment:
possibly add something like "this will be used as an exact match search, so be sure to test the search link for know issues. In the case of 3.14.0 that would be `https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+3.14.0%22`"

Main concern is that if some one does `affects:3.14` that wouldn't work b/c we expect `Affects: 3.14.0`.  But not that big of a concern!  Will approve PR and leave it up to you.

Reply:
True! Good call.